frontend/src/metabase/search/analytics.ts

```diff
@@ -0,0 +1,35 @@
+import { trackSchemaEvent } from "metabase/lib/analytics";
+import type { URLSearchFilterQueryParams } from "metabase/search/types";
+
+export const trackSearchEvents = ({
+  searchFilters,
+}: {
+  searchFilters: URLSearchFilterQueryParams;
+}) => {
+  let content_type = null;
+
+  if (searchFilters.type) {
```

Comment:
It's just my personal preference that I try to avoid reassignments at all. These few lines of code could be replaced by a function call. I found it feel uneasy to read `let` because I need to always be aware of when it could be reassigned. But a `const` guarantees that the value is only assigned here once.

```ts
const contentType = getContentType(searchFilters.type)
...
function getContentType(type) {
  if (type) {
    returns Array.isArray(type)? type: [type]
  }

  return null
}
```

what do you think?

Reply:
Understandable - I kept it in the function because I didn't feel it was "function-worthy" in terms of length and usage in any other part of the code. But I think it still looks nicer so I'll change it